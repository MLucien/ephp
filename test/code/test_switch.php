<?php

for ($i=0; $i<5; $i++) {
    switch ($i) {
        case 4:
            print "4 is greater than ";
        case 2:
            print "2 is greater than ";
        case 0:
            print "0... even numbers\n";
            break;
        default:
            print "$i is odd\n";
    }
}

