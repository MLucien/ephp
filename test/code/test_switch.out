0... even numbers
1 is odd
2 is greater than 0... even numbers
3 is odd
4 is greater than 2 is greater than 0... even numbers
